### DISCUSSION TITLE
New Database supports

### DISCUSSION THREAD

**Discussion by Aesth3tical** [reflex-dev/reflex#401] [General]
In the future will there be built in support for connections to other remote databases, i.e but not limited to MongoDB? That would be cool for folks who don't generally prefer SQL formatting, something to auto-migrate data between databases in the case someone switches remote databases might be cool as well perhaps.

**Comment by FHU-yezi** [comment]
In my option, we can support to add database connection features as plugins.

**Comment by Alek99** [comment]
Yes nice suggestion will look into this more, would be nice to support a broader range of db in the future
